static/elements/chromedash-process-overview.js

```diff
@@ -0,0 +1,71 @@
+import {LitElement, html} from 'lit-element';
+import {nothing} from 'lit-html';
+import '@polymer/iron-icon';
+import './chromedash-color-status';
+
+import style from '../css/elements/chromedash-process-overview.css';
+
+class ChromedashProcessOverview extends LitElement {
+  static styles = style;
+
+  static get properties() {
+    return {
+      feature: {type: Object},
+      process: {type: Array},
+      progress: {type: Array},
+      revisit: {type: String},
+      start: {type: String},
+      preview: {type: String},
```

Comment:
Looks like these 3 are just a const, and used only once.
1. Maybe just use the string in the template instead of have a variable for it?
2. If we want to keep string consts as variables, put them before the `class` - `const REVISIT_STRING = 'Revisit';` etc.

Reply:
I removed these.  I had planned to use this element in on the new.html page with different words, but then I decided to leave it off.